test/test_helper.rb

```diff
@@ -2,6 +2,8 @@
 require_relative '../config/environment'
 require 'rails/test_help'
 
+include AuthHelper
```

Comment:
it should be included inside ActiveSupport::TestCase, not to the global namespace